dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -264,12 +260,24 @@ private boolean checkRequiredTaskSuccess(
     return true;
   }
 
-  private void logStatusSummary(SummaryPrinter summaryPrinter, TaskSetState state) {
+  private void logFinalSummary(
+      SummaryPrinter summaryPrinter,
+      TaskSetState state,
+      OptionalLong outputFileLength,
+      Stopwatch stopwatch,
+      String outputFileLocation) {
     summaryPrinter.printSummarySection(
         linePrinter -> {
-          for (TasksReport item : state.getTasksReports()) {
-            linePrinter.println("%s", item);
-          }
+          StringBuilder executionSummary = new StringBuilder("Dumper ");
+          outputFileLength.ifPresent(length -> executionSummary.append("wrote " + length + ", "));
```

Comment:
If my understanding is correct, there are two possible results assuming that no data was written:
- If a file was created, there will be a message similar to "Dumper wrote 0 bytes".
- If no files were created, Dumper will also have written no bytes, but there will be no message about it.

It would be best IMO to either never report writing 0 bytes or to always report it.

Reply:
I've changed the message, so that it always prints `Dumper wrote 0 bytes` in such a case.